### PR TITLE
DOC: Update license headers

### PR DESCRIPTION
**Pull request by pastewka**
This PR adds maintenance script that update the license headers with the information on who modified the file when, pulled from the git log.

### CODE REVIEW

**Comment by pastewka** [comment]
Hi @jameskermode - this is an attempt to make the author information a bit more egalitarian. Please have a look and see if you like it. I could not identify who "Montmorency" is, have a look at the `AUTHORS` file.

**Comment by jameskermode** [comment]
Thanks for doing this. Will check it carefully next week (I’m on holiday this week.)

**Review by jameskermode — CHANGES_REQUESTED** [review]
Looks great - apart from adding Henry's name I don't see any other errors or omissions. Thanks again for doing this.

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Andreas Klemenz (Fraunhofer IWM) <andreas.klemenz@iwm.fraunhofer.de>
+Wolfram G. Nöhring (U. Freiburg) <wnoehring@simnetpc68.imtek.privat>
+Wolfram G. Nöhring (U. Freiburg) <wnoehring@o2.box>
+Jonas Oldenstaedt (U. Freiburg) <joldenstaedt@gmail.com>
+Lars Pastewka (U. Freiburg) <lars.pastewka@imtek.uni-freiburg.de>
+Lars Pastewka (U. Freiburg) <lars.pastewka@iwm.fraunhofer.de>
+Lars Pastewka (U. Freiburg) <lars.pastewka@kit.edu>
+Punit Patel (Warwick U.) <p.patel@warwick.ac.uk>
+Thomas Reichenbach (Fraunhofer IWM) <thomas.reichenbach@iwm.fraunhofer.de>
+Montmorency <lamberh@tcd.ie>
```
[the remark this answers is not in the document]
Comment (pastewka) [reply AUTHORS]: Sounds good, I just added his name

**Comment by pastewka** [comment]
Hi @jameskermode - comments should be addressed

**Review by jameskermode — APPROVED** [review]

**Pull request merged**